string type
geometry_msgs/Polygon geometry
